### PR TITLE
Refactor MemberStudies submodel of Member

### PR DESCRIPTION
**Pull request by k911**
Todo:

- [x]  ~~Documentation, because previous `PR` didn't have one~~ Next `PR`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,25 +10,25 @@
     "lint": "eslint .",
     "show:notes": "notes",
     "db:drop": "mongo db --eval \"db.dropDatabase()\"",
-    "db:migrate": "cd db && for col_name in articles members; do mongoimport --jsonArray --db db --collection $col_name --drop --file $col_name.json; done",
+    "db:migrate": "cd db && mongoimport --jsonArray --db db --collection articles --drop --file articles.json && mongoimport --jsonArray --db db --collection members --drop --file members.json",
```
Comment (FieryCod) [inline package.json]: Have no idea why you've changed that line.

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,4 @@ node_modules
 .directory
 npm-debug.log*
 *.orig
+package-lock.json
```
Comment (FieryCod) [inline .gitignore]: ?? It should be definitely in the repo

File: package.json (hunk 1 above)
Comment (zrebcu411) [inline package.json]: I think so too. Previous version is more elegant

**Comment by FieryCod** [comment]
@k911 Delete `package-lock.json` from `.gitignore` and restore the `db:migrate` as it was. In general, this `PR` is really good and I see that it introduces some of the crucial changes.
When you change those lines which I've listed please merge this `PR` as soon as it possible, because I need some of changes you've made :+1: 
Propose documentation in the next `PR` please.

File: package.json (hunk 1 above)
Comment (k911) [inline package.json]: Of course, it just doesn't work on Windows and I thought we still support it.

File: .gitignore (hunk 2 above)
Comment (k911) [inline .gitignore]: Ok

**Pull request merged**